tests/test_trusted_metadata_set.py

```diff
@@ -249,13 +255,15 @@ def version_modifier(timestamp: Timestamp) -> None:
         with self.assertRaises(exceptions.ReplayedMetadataError):
             self.trusted_set.update_timestamp(self.metadata["timestamp"])
 
-    def test_update_timestamp_snapshot_ver_below_trusted_snapshot_ver(self):
-        def version_modifier(timestamp: Timestamp) -> None:
-            timestamp.version = 3
+    def test_update_timestamp_snapshot_ver_below_current(self):
+        def bump_snapshot_version(timestamp: Timestamp) -> None:
+            timestamp.meta["snapshot.json"].version = 2
+
+        # set current known snapshot.json version to 2
+        timestamp = self.modify_metadata("timestamp", bump_snapshot_version)
+        self._root_updated_and_update_timestamp(timestamp)
```

Comment:
I see I made a mistake when writing this test and test_update_timestamp_new_timestamp_ver_below_trusted_ver test.
It must have been a mistake while rebasing.
Great catch!